src/Components/Header/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Container from "@mui/material/Container";
+import Typography from "@mui/material/Typography";
+
+const Header = () => {
+  return (
+    <React.Fragment>
+      <Container
+        sx={{
+          bgcolor: "rgb(80, 80, 80, 0.3)",
+          height: "40vh",
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+        }}
```

Comment:
You cannot use inline css we discussed about this in the last project, you cannot be making same mistakes